src/content/1.7/scale/webserver.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Optimize your webserver
+alias:
+- 1.7/optimizations/
+---
+
+# Optimize your webserver
+
+We often talk about optimizing the application side but less about the webserver.
+
+Which is a shame as there is a lot to gain from some simple configurations parameters. And you don't need to have hundred of visitors for them to be efficient, either to reduce your server's load or speed up your visitor's browsing.
+
+As we'll see, the configuration options we'll tweak are quite easy and should allow some nice improvements.
+
+# Webserver's function
+
+You may already be aware of, a webserver is a piece of software dedicated to satisfy world wide web requests.
```

Comment:
Feel free to link to external resources for people who would like to discover this topic or learn more.
For example here you can link to https://en.wikipedia.org/wiki/Web_server

Reply:
Added link.